src/Xablu.WebApiClient/WebApiClient.cs

```diff
@@ -22,12 +22,10 @@ public class WebApiClient : IWebApiClient
         private Lazy<HttpClient> _background;
         private Lazy<HttpClient> _userInitiated;
         private Lazy<HttpClient> _speculative;
-        private Func<HttpMessageHandler> httpHandler;
+        private Func<HttpMessageHandler> httpHandler = () => new HttpClientHandler();
```

Comment:
Just nit picking but I suggests to prefix private class fields with an underscore (so rename "httpHandler" to "_httpHandler"), so it matches with the other private fields (same convention).